modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -5549,6 +5622,161 @@ public void testGetScopeScopeKeyObjectEntriesPage() throws Exception {
 			_siteScopedObjectEntry1.getObjectEntryId());
 	}
 
+	@Test
+	public void testLocalizedObjectFieldWithAcceptLanguage() throws Exception {
```

Comment:
Just to discuss it, should be right to have a new java class called for `ObjectEntryGraphQLTest` just in order to have separated test from GraphQL and REST API? cc/ @4lejandrito @ccorreagg

Reply:
GrpahQL tests are autogenerated in the resource test (see [an example](https://github.com/liferay/liferay-portal/blob/63ab7283c1cdad7bff6f77f8e766ad33c8f6e566/modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-order-test/src/testIntegration/java/com/liferay/headless/commerce/admin/order/resource/v1_0/test/BaseChannelResourceTestCase.java#L556-L602)). Just rename the test to match that pattern.

Ideally the test should use the same GraphQLField classes and maybe we could place them in the HTTPTestUtil class... but we can do this at another time.